tests/test.env

```diff
@@ -5,3 +5,4 @@ export SAMPLEARCH=x86
 export ROSA=false
 export TEST_GROUP=full
 export MUSTGATHERTAG=2.3
+export IPV6=false
```

Comment:
@fjglira I think you need to assign the value of IPV6 from ${IPV6:=false} . Otherwise, it will always be false and you can't get the value set from the Dockerfile env.